FortranCon2021-stdlib/README.md

```diff
@@ -0,0 +1,41 @@
+# FortranCon2021-stdlib
+
+**Authors**:
+[Nathaniel Shaffer](https://github.com/nshaffer) (University of Rochester), 
+[Gabriel Brown](https://github.com/ghbrown) (University of Illinois Urbana-Champagne), 
+[Ondřej Čertik](https://github.com/certik) (Los Alamos National Laboratory),
+[William Clodius](https://githiub.com/wclodius2),
+[Robert Cohn](https://github.com/rscohn2) (Intel Corporation)
+[Milan Curcic](https://github.com/milancuric) (University of Miami),
+[Laurence Kedward](https://github.com/lkedward) (University of Bristol),
+[Sebastian Ehlert](https://github.com/awvwgk) (University of Bonn),
+[gareth-nx ???](https://github.com/gareth-nx),
+[Aman Godara](https://github.com/Aman-Godara),
+[Jing ???](https://github.com/Jim-215-Fisher),
+[Chetan Karwa](https://github.com/chetankarwa),
+[Arjen Markus](https://github.com/arjenmarkus),
+[Ian Giestas Pauli](https://github.com/14NGiestas) (São Paolo University),
+[Wileam Phan](https://github.com/wyphan) (University of Tennessee, Knoxville), 
```

Comment:
I only contributed a minor fix for CMake, so please feel free to omit my name, for now. Looking forward to contribute more in the future, though!
```suggestion
```